### PR TITLE
Initialpush

### PR DESCRIPTION
**Pull request by rberg2**
Initial push of sawtooth ansible roles.

### CODE REVIEW

File: roles/addrepo/tasks/main.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+- name: "Download Sawtooth Repo Key"
+  apt_key:
+    keyserver: keyserver.ubuntu.com
+    id: "{{ sawtooth_repo_key }}"
+
+- name: "Add Sawtooth Repo"
+  apt_repository:
+    repo: "deb http://repo.sawtooth.me/{{ sawtooth_repo }} xenial universe"
```
Comment (askmish) [inline roles/addrepo/tasks/main.yaml:9]: `[arch=amd64]`  should be specified in the repo def.

File: configs/config-example.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+# Max log file size
+maxbytes: 50000000
+# Number of files to keep
+backupcount: 20
+
+###### SGX configuration ######
+
+# spid: 'DEADBEEF00000000DEADBEEF00000000'
+# spid_cert_file: 'sgx-certificate.pem'
```
Comment (askmish) [inline configs/config-example.yaml:51]: Should that just be the filename or path to the pem file.

**Review by rbuysse — APPROVED** [review]

File: roles/addrepo/tasks/main.yaml (hunk 1 above)
Comment (rberg2) [reply roles/addrepo/tasks/main.yaml:9]: 👍  Please note I closed this PR and opened #2 due to messing up an interactive rebase and squashing the wrong commits together. This change will land in the other PR. Thanks!

File: configs/config-example.yaml (hunk 2 above)
Comment (rberg2) [reply configs/config-example.yaml:51]: The idea was that you would put the pem file in sawtooth-ansible/roles/sawtooth-poet-sgx/files and add the file name here, then Ansible will ensure the file is installed on every system and the configuration files updated. I will make a note of that in the example config.

**Pull request closed without merging**